deps/rabbit/src/rabbit_queue_consumers.erl

```diff
@@ -168,6 +168,8 @@ add(ChPid, CTag, NoAck, LimiterPid, LimiterActive,
                                                            delivery_count = InitialDeliveryCount}}}
         end,
     update_ch_record(C),
+    persistent_term:put(unsent_message_limit,
+        application:get_env(rabbit, classic_queue_consumer_unsent_message_limit, 200)),
```

Comment:
Not very important, but would it be better to place this into `rabbit_queue_consumers:new/0` instead because that's easier to reason about and called less often.

Reply:
Can we define the default of `200` at only one place?
Either here or in the Makefile.
I know that other RabbitMQ settings also define a default value 2 times (1 in the Makefile and 1 in the code as done here), but I never understood why that's done.
FWIW, I prefer this setting to not be exposed in `rabbitmq.conf` and only to bet set in `advanced.config`. This way we can get rid of the Makefile changes entirely including the Makefile default setting.

Reply:
I think that commonly set configuration goes into `rabbitmq.conf` which requires writing Cuttlefish translations.
Rarely or advanced config goes - as the name suggests - into `advanced.config` file.
This setting, given it has never been configurable, is for me a rare / advanced configuration.
Other internal credit flow settings such as `msg_store_credit_disc_bound` or `credit_flow_default_credit` are not exposed either in `rabbitmq.conf`.

Reply:
@ansd using `rabbit_queue_consumers:new/0` wont work here. We want this to be dynamic and configurable in real time, when new consumers attach to a queue. Which is why we're adding to `rabbit_queue_consumers:add/9`. This suggestion breaks the purpose of the change - we want capability to "accelerate" consumers on existing queues.

Reply:
@ansd credit-flow configs use `advanced.config` solely because they're configured as native Erlang terms/tuples. This config is just a simple integer, hence must be set in `rabbitmq.conf`. Which is much easier for operators to update/manage this config. Not sure I get the value/advantage of making configuring this unnecessarily more complex for users?

Reply:
> We want this to be dynamic and configurable in real time, when new consumers attach to a queue.

When exactly would you increase or decrease this value dynamically at runtime globally for all consumers running on this node?